MaplePacketLib2/Tools/PacketWriter.cs

```diff
@@ -12,9 +12,9 @@ public class PacketWriter : Packet {
             this.Length = 0;
         }
 
-        public static PacketWriter Of(ushort opcode, int size = DEFAULT_SIZE) {
+        public static PacketWriter Of(Object opcode, int size = DEFAULT_SIZE) {
             var packet = new PacketWriter(size);
-            packet.WriteUShort(opcode);
+            packet.WriteUShort((ushort) Convert.ToInt32(opcode));
```

Comment:
We shouldn't need to convert to an int anymore should we?

Reply:
Oh yeah, guess not, updated